src/components/network/NetworkGraph.vue

```diff
@@ -307,7 +307,7 @@ export default Vue.extend({
         state.network.view.selectedConnection,
       ],
       () => {
-        state.graph.updateNetworkGraph();
+        // state.graph.updateNetworkGraph();
```

Comment:
Do we need this line? Or could it be removed completely?

Reply:
Lets keep this line.
I checked that it reduces double updating of network graph.
But I am not sure if it is resolved in all situations.

Reply:
Ok :+1: